src/html_parser/readability.py

```diff
@@ -49,7 +49,7 @@ def parse(self, input: ParserInput) -> ParserOutput:
             )
         except Exception as e:
             logger.exception(
-                f"Could not fetch {input.document_source_url} for {input.document_id}"
+                f"Could not fetch {input.document_source_url} for {input.document_id}. Exception: {e}"
```

Comment:
This `Exception: {e}` bit isn't really required - `logger.exception()` includes the full exception context